broker/plugins/kubejobs/plugin.py

```diff
@@ -233,6 +235,59 @@ def errors(self):
             return ()
         return self.rds.lrange("job:errors", 0, -1)
 
+    def validate(self, data):
+        data_model = {
+            "cmd": list,
+            "control_parameters": dict,
+            "control_plugin": six.string_types,
+            "env_vars": dict,
+            "img": six.string_types,
+            "init_size": int,
+            "monitor_info": dict,
+            "monitor_plugin": six.string_types,
+            "redis_workload": six.string_types,
+            "enable_visualizer": bool
+            # The parameters below are only needed if enable_visualizer is True
```

Comment:
nit: as there's no actual validation, this comment could go away or be moved to somewhere else within the method.

Reply:
There is a validation, but not in the main data model 'for loop', its only made if the enable visualizer is set to true, the comments here are to be visually clearer to the reader that they do are validated, even tho they are not in data model